app/controllers/concerns/member_helper.rb

```diff
@@ -56,21 +56,29 @@ def user_ids_for_new_members(member_params)
   def invite_new_users(user_ids, send_notification: true)
     user_ids.filter_map do |id|
       if id.present? && (id.to_i == 0 || EmailValidator.valid?(id)) # we've got an email - invite that user
```

Comment:
Not related to this PR, but emails are allowed to start with numbers. So the `id.to_i` will return that number when the email starts with one.


```ruby
"123max.mustermann@example.com".to_i # => 123
```

Maybe we should only check for the EmailValidator?